### PR TITLE
Winter Themed Spotlights 2017

### PR DESCRIPTION
**Pull request by Shadowthegod**
### Description
The article for the Winter Themed spotlights

### Status
Pending Review 

### Changelog
- added article
- fixed wrong markdown

### CODE REVIEW

File: news/2017-12-24-winter-themed-spotlights-2017.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+![](https://assets.ppy.sh/media/generic-header.png)
+
+With that said, we just released the Winter Themed Charts and can be found [here](https://osu.ppy.sh/p/chart?ch=WINCH17). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access.
+
+# Winter Themed Spotlights 2017
+
+## osu!
+
++ [Hiro - Carol Megamix](http://osu.ppy.sh/s/64901) by [LunarSakuya](https://osu.ppy.sh/u/1027860)
++ [Tonon - Shirayuki ~sirayuki~](http://osu.ppy.sh/s/429316) by [yf_bmp](https://osu.ppy.sh/u/1243669)
```
Comment (Xinnoh) [inline news/2017-12-24-winter-themed-spotlights-2017.md]: code breaks this

File: news/2017-12-24-winter-themed-spotlights-2017.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
++ [TOKIO - ding-dong](http://osu.ppy.sh/s/375452) by [Volta](https://osu.ppy.sh/u/4154071)
+
+## osu!catch
+
++ [Odd Nordstoga - Jul i Svingen](http://osu.ppy.sh/s/381074) by [Benny-](https://osu.ppy.sh/u/4023183)
++ [Comedian Harmonists - Stille Nacht, Heilige Nacht](https://osu.ppy.sh/s/534385) by [JBHyperion](https://osu.ppy.sh/u/4879508)
++ [Kugimiya Rie & Kitamura Eri - Holy Night](https://osu.ppy.sh/s/227585) by [Kurokami](https://osu.ppy.sh/u/260933)
++ [Mahito Yokota - White Snow](https://osu.ppy.sh/s/221167) by [Sey](https://osu.ppy.sh/u/553656) and [mingmichael](https://osu.ppy.sh/u/1570876)
++ [u's - Snow halation](https://osu.ppy.sh/s/209390) by [CLSW](https://osu.ppy.sh/u/531253) and [Eclipst](https://osu.ppy.sh/u/2083612)
++ [Chata - Winter Bells](https://osu.ppy.sh/s/696064) by [lit120](https://osu.ppy.sh/u/3109248)
```
Comment (Xinnoh) [inline news/2017-12-24-winter-themed-spotlights-2017.md]: by sinnoh

File: news/2017-12-24-winter-themed-spotlights-2017.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
++ [SOUND HOLIC Vs. dj TAKA feat. YURiCa - TIEFSEE](http://osu.ppy.sh/s/457878) by [Nardoxyribonucleic](https://osu.ppy.sh/u/876419)
++ [TOKIO - ding-dong](http://osu.ppy.sh/s/375452) by [Volta](https://osu.ppy.sh/u/4154071)
+
+## osu!catch
+
++ [Odd Nordstoga - Jul i Svingen](http://osu.ppy.sh/s/381074) by [Benny-](https://osu.ppy.sh/u/4023183)
++ [Comedian Harmonists - Stille Nacht, Heilige Nacht](https://osu.ppy.sh/s/534385) by [JBHyperion](https://osu.ppy.sh/u/4879508)
++ [Kugimiya Rie & Kitamura Eri - Holy Night](https://osu.ppy.sh/s/227585) by [Kurokami](https://osu.ppy.sh/u/260933)
++ [Mahito Yokota - White Snow](https://osu.ppy.sh/s/221167) by [Sey](https://osu.ppy.sh/u/553656) and [mingmichael](https://osu.ppy.sh/u/1570876)
++ [u's - Snow halation](https://osu.ppy.sh/s/209390) by [CLSW](https://osu.ppy.sh/u/531253) and [Eclipst](https://osu.ppy.sh/u/2083612)
```
Comment (Xinnoh) [inline news/2017-12-24-winter-themed-spotlights-2017.md]: Eclipst -> Vhy

File: news/2017-12-24-winter-themed-spotlights-2017.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## osu!mania
+
++ [BeautifulDay - Christmas Memories](https://osu.ppy.sh/s/241103) by [arviejhay](https://osu.ppy.sh/u/145871)
++ [Tamura Yukari - MERRY MERRY MERRY MENU... Ne!](http://osu.ppy.sh/s/536964) by [hannanos](https://osu.ppy.sh/u/2847824)
++ [Mitchie M feat. Hatsune Miku - Suki! Yuki! Maji Magic](http://osu.ppy.sh/s/682665) by [Sinnoh](https://osu.ppy.sh/u/4236057)
++ [Ti7 - Love Maker](http://osu.ppy.sh/s/382365) by [DJPop](https://osu.ppy.sh/u/2363)
+
+**The Winter Themed Spotlight 2017 will end on the 31st January 2018.**
+
+We wish you a merry christmas and a happy new year with full of top ranks!
```
Comment (Xinnoh) [inline news/2017-12-24-winter-themed-spotlights-2017.md]: remove with

File: news/2017-12-24-winter-themed-spotlights-2017.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Winter Themed Spotlights 2017"
+date: 2017-12-24 13:31:35 +0000
+---
+
+Greetings everyone! We hope all of you have a wonderful and snowy Christmas today and celebrate it with loved ones. For this special day, the Spotlights Team prepared a winter themed spotlight for those who feel like only presents are just not enough. Let's drink some hot tea at this special day while clicking those circles; hitting the drum; catching those healthy fruits and go octopus-mode with the squares!
```
Comment (Xinnoh) [inline news/2017-12-24-winter-themed-spotlights-2017.md]: replace tea with hot chocolate maybe, more festive?

File: news/2017-12-24-winter-themed-spotlights-2017.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+With that said, we just released the Winter Themed Charts and can be found [here](https://osu.ppy.sh/p/chart?ch=WINCH17). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access.
+
+# Winter Themed Spotlights 2017
+
+## osu!
```
Comment (Xinnoh) [inline news/2017-12-24-winter-themed-spotlights-2017.md]: where's voli's map D:

File: news/2017-12-24-winter-themed-spotlights-2017.md (hunk 1 above)
Comment (Shadowthegod) [reply news/2017-12-24-winter-themed-spotlights-2017.md]: Hopefully this is fixed now. o.O

File: news/2017-12-24-winter-themed-spotlights-2017.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Winter Themed Spotlights 2017"
+date: 2017-12-24 13:31:35 +0000
+---
+
+Greetings everyone! We hope all of you have a wonderful and snowy Christmas today and celebrate it with loved ones. For this special day, the Spotlights Team prepared a winter themed spotlight for those who feel like only presents are just not enough. Let's drink some hot chocolate at this special day while clicking those circles; hitting the drum; catching those healthy fruits and go octopus-mode with the squares!
```
Comment (MegaApplePi) [inline news/2017-12-24-winter-themed-spotlights-2017.md]: was there a reason to split the list with semicolons (you'd only use semicolons in lists for when the list is too complicated with commas)

**Comment by Ephemeralis** [comment]
stale and defunct

**Pull request closed without merging**